tests/unit/components/PageFooter.spec.js

```diff
@@ -34,4 +34,15 @@ describe('components/search/PageFooter', () => {
     const renderedList = wrapper.findAll('footer ul li');
     renderedList.length.should.eq(2);
   });
+
+  it('contains the language selector', () => {
+    const wrapper = factory();
+    wrapper.setProps({
+      enableLanguageSelector: true
+    });
+    const selector = wrapper.find('[data-qa="language selector"]');
+
+    selector.isVisible().should.equal(true);
+  });
+  
```

Comment:
this raises a linting error at the start of the unit tests
```
 Module Warning (from ./node_modules/@vue/cli-plugin-eslint/node_modules/eslint-loader/index.js):
  error: Trailing spaces not allowed (no-trailing-spaces) at tests/unit/components/PageFooter.spec.js:47:1:
    45 |     selector.isVisible().should.equal(true);
    46 |   });
  > 47 |   
       | ^
    48 | });
    49 | 

```

Reply:
That has been removed, @lbiedinger